Framework/Geometry/test/MeshObjectTest.h

```diff
@@ -50,21 +50,238 @@ class MockRNG final : public Mantid::Kernel::PseudoRandomNumberGenerator {
   MOCK_CONST_METHOD0(max, double());
   GCC_DIAG_ON_SUGGEST_OVERRIDE
 };
+
+boost::shared_ptr<MeshObject> createCube(const double size, const V3D &centre) {
```

Comment:
👍 for making these free functions.

The return values should not be wrapped as `boost::shared_ptr` unless there really is sharing happening, and as far as I can tell, there isn't. `std::unique_ptr` would be a much better choice for return value for these methods.